Source/WebCore/svg/SVGTransformList.cpp

```diff
@@ -62,54 +62,75 @@ AffineTransform SVGTransformList::concatenate() const
     return result;
 }
 
-template<typename CharacterType> bool SVGTransformList::parseGeneric(StringParsingBuffer<CharacterType>& buffer)
+template<typename CharacterType> bool SVGTransformList::parseGeneric(StringParsingBuffer<CharacterType>& buffer, ListReplacement listReplacement)
 {
     bool delimParsed = false;
     skipOptionalSVGSpaces(buffer);
 
-    while (buffer.hasCharactersRemaining()) {
-        delimParsed = false;
-        
-        auto parsedTransformType = SVGTransformable::parseTransformType(buffer);
-        if (!parsedTransformType)
-            return false;
+    auto parsingSucceeded = [&]() {
```

Comment:
If calling `clearItems();` was moved back to `SVGTransformList::parse(StringView value)`, there is no need for put the parsing code in a lambda.